src/current/v24.1/molt-fetch.md

```diff
@@ -99,6 +99,44 @@ Complete the following items before using MOLT Fetch:
 
 - If a MySQL database is set as a [source](#source-and-target-databases), the [`--table-concurrency`](#global-flags) and [`--export-concurrency`](#global-flags) flags **cannot** be set above `1`. If these values are changed, MOLT Fetch returns an error. This guarantees consistency when moving data from MySQL, due to MySQL limitations. MySQL data is migrated to CockroachDB one table and shard at a time, using [`WITH CONSISTENT SNAPSHOT`](https://dev.mysql.com/doc/refman/8.0/en/commit.html) transactions.
 
+## Security
+
+Cockroach Labs **strongly** recommends the following:
+
+### Secure connections
+
+- Use secure connections to the source and target database whenever possible.
+- By default, insecure connections (i.e., `sslmode=disable` on PostgreSQL; `sslmode` not set on MySQL) are disallowed. When using an insecure connection, `molt fetch` returns an error. To override this check, you can enable the `--allow-tls-mode-disable` flag. Do this **only** if a secure SSL/TLS connection to the source or target database is not possible.
+
+### Connection strings
+
+- Avoid plaintext connection strings.
+- Provide your connection strings as environment variables. 
+- If possible within your security infrastructure, use an external secrets manager to load the environment variables from stored secrets.
+
+	For example, to export connection strings as environment variables:
+
+	~~~ shell
+	export SOURCE="postgres://postgres:postgres@localhost:5432/molt?sslmode=disable"
+	export TARGET="postgres://root@localhost:26257/molt?sslmode=disable"
+	~~~
+
+	Afterward, to pass the environment variables in `molt fetch` commands:
+
+	~~~ shell
+	molt fetch \
+	--source "$SOURCE" \
+	--target "$TARGET" \
+	--table-filter 'employees' \
+	--bucket-path 's3://molt-test' \
+	--table-handling truncate-if-exists
+	~~~
+
+### Secure cloud storage
+
+- When using a [cloud bucket](#cloud-storage) for your intermediate store, ensure that access control is properly configured. Refer to the [GCP](https://cloud.google.com/storage/docs/access-control) or [AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-iam.html) documentation.
+- Do not use public cloud storage for production.
```

Comment:
I think it's more accurate to say: `Do not use a public cloud storage bucket in production.`

Reply:
`Ideally, test buckets should also be locked down to the proper permissions.`